packages/api-server/api_server/routes/internal.py

```diff
@@ -134,13 +135,18 @@ async def process_msg(msg: Dict[str, Any], fleet_repo: FleetRepository) -> None:
 
 
 @router.websocket("")
-async def rmf_gateway(websocket: WebSocket):
+async def rmf_gateway(
+    websocket: WebSocket, logger: LoggerAdapter = Depends(get_logger)
+):
+    connection_manager = ConnectionManager(logger)
```

Comment:
correct me if I'm wrong, `rmf_gateway(...)` will be called for every new websocket connection request, is that right?
In that case, moving `ConnectionManager` into `rmf_gateway(...)`, will just make it track a single websocket connection, so it will never be able to keep track multiple websocket connections as intended

Reply:
Yeah, I reverted it to be a global, the downside is that we no longer get extra logging info.

Reply:
yeah I think that's fine for now, thanks!